docs/SupportedCommands/Commands.md

```diff
@@ -78,7 +78,10 @@ Examples (Ruby):
 | `hasFocus`                 | 					bool	 				       | 					4.5.1+	 				|
 | `id`                       | 					string			         | 					4.5.1+	 				|
 | `isDisplayed`              | 					bool	 				       | 					4.5.1+	 				|
+| `isFullyDisplayed`               | 					bool	 				       | 					4.6.0+	 				|
 | `isHittable`               | 					bool	 				       | 					4.5.1+	 				|
+| `isHorizontalScrolling`               | 					bool	 				       | 					4.6.0+	 				|
```

Comment:
4.5.2?

Reply:
No, it will be in 4.6+ because that what develop is forked to right now... Could be cherry-picked though

Reply:
I would cherry-pick to 4.5.2